.gitignore

```diff
@@ -11,6 +10,10 @@ db/
 .vscode
 .venv
 
+.coverage.*
```

Comment:
It's best if PRs focus on one single piece of functionality or bug fix - can you move this to another PR?

Reply:
Sure. I don't remember this being an issue so I'll just remove for now unless it comes back up.

Reply:
Done.